assets/solution/deployment.template.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema-template": "1.0.0",
+  "$schema-template": "2.0.0",
```

Comment:
Is this version related to the schemastore json schema version? Now our latest version is 1.1.0

Reply:
Discussed offline. Schemastore will have a new version.